client/client-lib/src/api.rs

```diff
@@ -33,11 +32,10 @@ pub trait LnFederationApi {
     async fn offer_exists(&self, payment_hash: Sha256Hash) -> FederationResult<bool>;
 }
 
-#[cfg_attr(target_family = "wasm", async_trait(? Send))]
-#[cfg_attr(not(target_family = "wasm"), async_trait)]
+#[apply(async_trait_maybe_send!)]
```

Comment:
Is `apply` something we're going to use for other stuff? If not maybe something like:

```
#[async_trait_ext(maybe_send)]
```

BTW. That `!` is easy to overlook and not clear (is it a `!Send` or some macro invocation)? If we need "not" in there, maybe let's include it as `maybe_not_send`?

Reply:
I see. So `apply` is from some 3rd party library, and then `async_trait_maybe_send!` is a macro invocation. 

Oh well. Will learn how to live it it.

Reply:
It is a tradeoff, the definition of macro is really simple. but the usage is a bit weird.

I can write a proc macro attribute if you want.

Reply:
Nah, good enough.